release-notes/Cube/Cube_Feature_Release_10.5/Cube_10.5.5.md

```diff
@@ -12,49 +12,130 @@ uid: Cube_Feature_Release_10.5.5
+![Topology app](~/user-guide/images/EPMIntegration_Topology_app.png)
 
-#### EPM functionality is now fully integrated in DataMiner Cube [ID 42221]
+To make sure the Topology app is available, the following configuration is needed:
 
-<!-- MR 10.4.0 [CU14] / 10.5.0 [CU2] - FR 10.5.5 -->
+1. Make sure the DMS contains at least one front-end EPM Manager element.
+
+   For DataMiner Cube to detect these elements, they should be active, the *type* attribute of the *Display* tag in their connector should be set to "element manager", and they should contain a parameter named *ElementManagerType* set to 1. Also, their name should preferably start with "FE".
 
-All EPM functionality is now fully integrated in the UI of DataMiner Cube. Up to now, this required the *CPEIntegration* soft-launch option to be enabled.
+   Each front-end EPM Manager element will describe a topology chain and its associated filters.
 
-In the sidebar, you can now open the *Topology* pane, which allows you to select a topology chain, drill down to any of the topology level in that chain. On top of that, all EPM-related data can now also be accessed via the Surveyor or the Alarm Console.
+1. Go to *System Center* > *System settings* > *EPM config*, and add the front-end EPM Manager elements to the list.
 
-##### Configuration
+   In the Topology app, you will now be able to select the front-end managers and the corresponding topology chains.
 
-1. Make sure the DMS contains at least one EPM frontend manager element.
+##### EPM masking info<!-- 26002 -->
 
-   For DataMiner Cube to detect these elements, they should be active, the *type* attribute of the *Display* tag in their connector should be set to "element manager", and they should contain a parameter named *ElementManagerType* set to 1. Also, their name should preferably start with "FE".
+If an EPM object in the topology is masked, you can get information on who masked it and when via the *Masking info* option in the hamburger menu of the Topology app:
+
+![Masking info option](~/user-guide/images/EPMIntegration_Masking_info.png)
 
-   Each EPM frontend manager element will describe a topology chain and its associated filters.
+##### System Type and System Name properties
 
-1. Go to *System Center > System settings > EPM config*, and add all EPM frontend manager element(s) to the list.
+For much of the EPM integration functionality, the system type and system name must be configured in EPM Manager protocols. The system type is configured with the name attribute of the *Topologies.Topology.Cell* tag, and the system name is configured with the *columnPid* attribute of the *Topologies.Topology.Cell.Exposer.LinkedIds.LinkedId* tag. For example, in the configuration below, *Amplifier* is the system type, and the system name is indicated by the specified column PID:
```

Comment:
The System Name will be the value of a cell in the column with this pid once data is filled in, e.g. system type can be "Location", system name can be "Izegem" or "Brugge"